pstore/pstore.go

```diff
@@ -256,6 +256,10 @@ func (c *Consumer) flush() (err error) {
 			iter.Commit()
 		}
 	}
+	// Help out the GC
+	for i := range c.toBeCommitted {
+		c.toBeCommitted[i] = nil
+	}
 	// Clear out pending iterators
 	c.toBeCommitted = c.toBeCommitted[:0]
```

Comment:
I just noticed this PR is still open. I commented on 19-Jul but haven't seen an update.